drivers/include/xbee.h

```diff
@@ -109,7 +105,7 @@ typedef struct {
     ng_nettype_t proto;                 /**< protocol the interface speaks */
     uint8_t options;                    /**< options field */
     uint8_t addr_short[2];              /**< onw 802.15.4 short address */
-    uint8_t addr_long[8];               /**< own 802.15.4 long address */
+    ieee_802154_long_t addr_long;       /**< own 802.15.4 long address */
```

Comment:
*If* we have to introduce a type for that, I'd prefer `eui64_t`. It's shorter, more general and the IEEE 802.15.4 is exactly that.